Marlin/Marlin_main.cpp

```diff
@@ -163,6 +163,7 @@
  * M209 - Turn Automatic Retract Detection on/off: S<0|1> (For slicers that don't support G10/11). (Requires FWRETRACT)
           Every normal extrude-only move will be classified as retract depending on the direction.
  * M211 - Enable, Disable, and/or Report software endstops: S<0|1> (Requires MIN_SOFTWARE_ENDSTOPS or MAX_SOFTWARE_ENDSTOPS)
+ * M217 - Send text to serial output.  Text is preceeded by "//".
```

Comment:
I wasn't sure about the protocol for using a code which appeared to already be defined.  `M118` would have been my first choice too :-)